cmd/acra-translator/acra-translator.go

```diff
@@ -83,7 +83,10 @@ func main() {
 		os.Exit(1)
 	}
 
-	logging.Customize().SetFormat(*loggingFormat).SetServiceName(ServiceName).Complete()
+	// Start customizing logs here (directly after command line arguments parsing)
+	formatter := logging.CreateFormatter(*loggingFormat)
+	logging.SetServiceName(formatter, ServiceName)
+	log.SetOutput(os.Stderr)
```

Comment:
Similarly, **acra-translator** seems to be the only one without

```go
log.WithField("version", utils.VERSION).Infof("Starting service %v [pid=%v]", ServiceName, os.Getpid())
```

during startup.

Reply:
thank you